bids-validator/validators/microscopy/ometiff.js

```diff
@@ -77,10 +84,15 @@ const checkOptionalFields = (omePath, omeData, jsonData) => {
   }
 
   if (
-    omeData['OME']['Instrument'] &&
-    omeData['OME']['Instrument'][0]['Objective']
+    omeData[`${namespace}OME`][`${namespace}Instrument`] &&
+    omeData[`${namespace}OME`][`${namespace}Instrument`][0][
+      `${namespace}Objective`
+    ]
```

Comment:
```suggestion
    omeData[`${namespace}OME`][`${namespace}Instrument`][0][`${namespace}Objective`]
```
The formatting on 3 lines seems odd to me, can it be on 1 line?

Reply:
This format is suggested by eslint, it looks odd to me as well. I will update them then.